### DISCUSSION TITLE
Web UI won't open. Do I have a configuration issue?

### DISCUSSION THREAD

**Discussion by smizles** [hassio-addons/addon-tasmoadmin#188] [Q&A] [answered]
# Problem/Motivation

> Went to use the add-on for the first time, but the Web UI won't open (web UI's for other add-ons work fine).

## Expected behavior

> Clicking "Open Web UI" button should open the web GUI.

## Actual behavior

> Clicking "Open Web UI" button redirects to chrome's "This site can't be reached" placeholder page with the error "ERR_CONNECTION_REFUSED" listed.

## Steps to reproduce

> Install add-on in Home Assistant OS 6.2 (core-2021.8.5). Click "Start" button on the add-on page. Watch logs to wait for it to finish starting up. Click "Open Web UI" button.

## Proposed changes

> Fix the breaking change or stronger documentation to prevent configuration mistakes (if I made one)?

Installation/ start up log:
![Screenshot_20210810-152814_Home Assistant](https://user-images.githubusercontent.com/25838538/128922964-ce67031d-057d-4671-980c-9d32eb37844a.jpg)

**Comment by frenck** [comment, marked as answer]
Does it work when you are on your local network? Please note, this integration does not support ingress.

**Reply by smizles** [reply]
> Does it work when you are on your local network?

That was 100% of my problem.

I didn't see anything in the docs about not supporting ingress, but I tend to setup these Home Assistant extras late at night so that might be an oversight. However, there are definitely several "mystery steps" missing from the documentation that I took a leap of on (i.e. it took me a while to realize that the initial login screen is actually an initial credential setup screen - I spent 15 minutes plunking around the net looking for what the initial login credentials were supposed to be).

I would love to see this issue get moved to a feature request since I access my instance 99.9% of the time via my DuckDNS domain!

**Reply by smizles** [reply]
FYI @frenck, I tried clicking the "Mark as answer" button, but nothing happens. Is that because this was moved from Issues to Q&A?

**Comment by bbostock** [comment]
I have the same or similar problem with the WebUI loading. The Chrome browser now shows the error: "myid.duckdns.org uses an unsupported protocol." 
and another Chromium based browser shows:
"Unsupported protocol. The client and server don't support a common SSL protocol version or cipher suite."
It works with SSL turned off. 
This started after installing the August 8.6 update of both supervisor and core. 
It won't work with a local connection either.

**Comment by 0xa51f** [comment]
Same problem as @bbostock, i use duckdns and chrome normally

using the SSL and the IP:9541 on LAN on Firefox gives:
```

      Secure Connection Failed
      
      An error occurred during a connection to 192.168.X.X9541. Cannot communicate securely with peer: no common encryption algorithm(s).
      
      Error code: SSL_ERROR_NO_CYPHER_OVERLAP
      
      The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
```

With ssl: false the page works

**Comment by charettepa** [comment]
I can load the page with SSL on 
but I cant ever upgrade the firmware on any devices
OTA is set to SSL on as well

with SSL off and OTA set to SSL off (unchecked)
I can perform firmware updates
but then I cant load tasmoadmin in iframe as its http

any chance tasmoadmin can get ingess ?
